contracts/utils/GameOwner.sol

```diff
@@ -10,25 +10,30 @@ contract GameOwner is Ownable {
 
     using Address for address;
 
+    event SetGameOwnerEvent(address indexed gameOwner);
+
     /**
      * Constructor method which calls initial setters for all the contracts
      */
     constructor(address _gameOwnerAddress) {
+        require(_gameOwnerAddress != address(0), "GameOwner: game owner address is 0x0");
         gameOwnerAddress = _gameOwnerAddress;
     }
 
     /**
      * Setter method for gameOwnerAddress variable
      */
-    function setGameOwnerAddress(address _newAddress) public onlyOwner {
+    function setGameOwnerAddress(address _newAddress) external onlyOwner {
+        require(_newAddress != address(0), "GameOwner: game owner address is 0x0");
```

Comment:
"GameOwner: game owner address can't be 0x0"

Reply:
Fixed